src/pages/NewUser/NewUser.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC, useState, useEffect } from 'react'
+import { UniversalForm, UniversalModal } from '@/components'
+import { ActionFunctionArgs, redirect, useNavigate } from 'react-router-dom'
+import { newUserFormProps } from './helper'
+import { ADMIN_ROUTES_NAMES } from '@/routing/routes.names'
+import { NewUserProps } from './helper'
+import { createNewUser, createNewUserWittEmailAndPassword } from '@/firebase/firestore'
+import classes from './NewUser.module.scss'
+
+export const addNewUser = async ({ request }: ActionFunctionArgs) => {
```

Comment:
is this one really should be here? I guess we discussed that it probably should be somewhere in routes configuration

Reply:
addNewUser is replaced to 'helper' file